services/src/main/java/org/keycloak/protocol/oidc/endpoints/TokenIntrospectionEndpoint.java

```diff
@@ -148,6 +151,15 @@ private void checkRealm() {
         }
     }
 
+
+    private void checkParameterDuplicated(MultivaluedMap<String, String> formParams) {
+        for (String key : formParams.keySet()) {
+            if (formParams.get(key).size() != 1) {
+                throw new ErrorResponseException(Errors.INVALID_REQUEST, "duplicated parameter", Status.BAD_REQUEST);
```

Comment:
One details: Could you use "throwErrorResponseException" method to make sure that also event is sent?